README.md

```diff
@@ -15,7 +15,7 @@ This page is available as an easy-to-read website at [https://ebookfoundation.gi
 
 This list was originally a clone of [StackOverflow - List of Freely Available Programming Books](https://web.archive.org/web/20140606191453/http://stackoverflow.com/questions/194812/list-of-freely-available-programming-books/392926) with contributions from Karan Bhangui and George Stocker.
 
-The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has grown to become one of [GitHub's most popular repositories](https://octoverse.github.com/), with 210,000+ stars, 6800+ commits, 1900+ contributors, and 45,000+ forks.
+The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has grown to become one of [GitHub's most popular repositories](https://octoverse.github.com/), with 221,000+ stars, 6900+ commits, 1900+ contributors, and 47,100+ forks.
```

Comment:
If we're going to use properly formatted numbers, probably best to be consistent and use it for all of them.

```suggestion
The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has grown to become one of [GitHub's most popular repositories](https://octoverse.github.com/), with 221,000+ stars, 6,900+ commits, 1900+ contributors, and 47,100+ forks.
```

Reply:
nice format typo catch. Applying...

If I'd have to choose one, will be without commas.